source/event_hub.ts

```diff
@@ -247,7 +237,7 @@ export class EventHub {
    *
    * If timeout is non-zero, the promise will be rejected if the event is not
    * sent before the timeout is reached. Should be specified as seconds and
-   * will default to 10.
+   * will default to 30.
```

Comment:
Isn't this a breaking change?

Reply:
It was simply my noticing that the comment was incorrect.

Reply:
Ah okay. All good then!